src/utils/backend.js

```diff
@@ -2,7 +2,7 @@
 import store from '../store';
 import { wrapTry } from './index';
 
-const backendFetch = (path, ...args) => wrapTry(
+export const backendFetch = (path, ...args) => wrapTry(
```

Comment:
Please define `errorreport` method here, instead of exporting `backendFetch`. Later we will replace all of them with API generated by swagger file.